### PR TITLE
Version 1

### PR DESCRIPTION
**Pull request by GusPrice**
Introduce version 1 for the module to create Honeycomb Refinery and related infrastructure in GCP.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,50 @@
-# terraform-honeycomb-refinery
\ No newline at end of file
+# terraform-honeycomb-refinery
+
+This module allows you to create a Managed Instance Group running Honeycomb Refinery and a single compute instance running a small Redis server to support the Refinery instances.
```
Comment (softwarebygabe) [inline README.md]: `Honeycomb Refinery` could be hyperlink to something?

File: redis.tf [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+Use of this source code is governed by a MIT license that can be found in the LICENSE file.
+
+*/
+
+module "gce_container_redis" {
+  source  = "terraform-google-modules/container-vm/google"
+  version = "3.1.0"
+
+  container = {
+    image = "us-docker.pkg.dev/ritten-ops/external/redis:7.2"
```
Comment (softwarebygabe) [inline redis.tf]: should this be changed?

File: refinery.tf [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+  startup_script = templatefile("${path.module}/config/startup.sh.tpl", { config_path = local.refinery_config_path, source_config_path = local.source_config_path, source_rules_path = local.source_rules_path })
+  service_account = {
+    email  = google_service_account.honeycomb_refinery.email
+    scopes = ["cloud-platform"]
+  }
+  tags = ["refinery"]
+
+  /* network */
+  subnetwork = data.google_compute_subnetwork.primary_subnetwork.id
+  # can_ip_forward = var.can_ip_forward
```
Comment (softwarebygabe) [inline refinery.tf]: rm?

File: redis.tf (hunk 2 above)
Comment (GusPrice) [reply redis.tf]: good catch, yes.

**Review by softwarebygabe — APPROVED** [review]

**Pull request merged**